app/controllers/admin/loan_questions_controller.rb

```diff
@@ -14,12 +14,17 @@ def new
     field_set = CustomFieldSet.find_by(internal_name: 'loan_' + field_set_name)
     @loan_question = field_set.custom_fields.build
     authorize @loan_question
-    @loan_type_options = Loan.loan_type_options
+    @loan_question.build_complete_requirements
     render_form
   end
 
   def edit
-    @loan_type_options = Loan.loan_type_options
+    @loan_question.build_complete_requirements
+    @requirements = @loan_question.custom_field_requirements
+    # TODO: Restore @requirements to item similar to below
+    # Using this line results in the following error for the below line:
+      # undefined method `label' for nil:NilClass:
+    # @requirements = @loan_question.custom_field_requirements.sort_by { |i| i.loan_type.label.text }
```

Comment:
@linesarefuzzy: This line breaks when used. It may be that some of the Custom Field Requirements do not have an option. I worked around this by checking for `sf.object.option_id` in the view. But this logic should be moved out of the view.